src/03-modality-agnostic-files.md

```diff
@@ -251,6 +251,71 @@ to date of birth.
 }
 ```
 
+## Samples file
+
+Template:
+
+```Text
+samples.tsv
+samples.json
+```
+
+The purpose of this RECOMMENDED file is to describe properties of samples
+when the `sample-<label>` key/value pair is used in filenames.
```

Comment:
I agree with @satra that this should be REQUIRED if `sample` is present. Something like the following?

```suggestion
The purpose of this file is to describe properties of samples, indicated by the `sample` entity.
This file is REQUIRED if `sample-<label>` is present in any file name within the dataset.
```

Reply:
I went with “recommended” because I think all the other `tsv` files in the specs are either recommended or optional, even `participants.tsv`. But I agree with @satra that the `sample_type` column is reason enough to make the file REQUIRED, I was just wondering if that makes sense with the rest of the spec? If so, I agree with the suggested changes.

Reply:
participants.tsv is not REQUIRED because there is no required metadata in it. if `sample_type` must exist for a dataset to be interpretable, then this should be REQUIRED.